src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceAccounts.java

```diff
@@ -0,0 +1,214 @@
+        button.click();
+    }
+
+    /**
+     * Method to click button.
+     * @param button to click.
+     */
+    public void clickSaveNewAccountButton(final WebElement button) {
+        button.click();
+    }
+
+    /**
+     * Method to set accountName attribute.
+     * @param accountName string value.
+     */
+    public void setAccountNameTextField(final String accountName) {
+        webDriver.findElement(this.accountNameTextField).sendKeys(accountName);
+    }
+
+    /**
+     * Method to know if the page of new account is displayed.
+     * @return true or false.
+     */
+    public boolean isNewAccountButtonDisplayed() {
+        return this.webDriver.findElement(this.newAccountButton).isDisplayed();
+    }
+
+    /**
+     * Method to create a new account.
+     * @param accountName the account name.
+     */
+    public void createNewAccount(final String accountName) {
+        WebElement button = getNewAccountButton();
+        clickNewAccountButton(button);
+        setAccountNameTextField(accountName);
+    }
+
+    /**
+     * Method to know if an account is saved.
+     * @return the new account name displayed in label.
+     */
+    public String newAccountSavedName() {
+        return this.webDriver.findElement(this.newAccountLabel).getText();
+    }
+
+    /**
+     * Method to set view in accounts list.
+     * @param accountView view list.
+     */
+    public void setAccountsViewList(final String accountView) {
+        Select select = new Select(this.webDriver.findElement(this.accountsViewList));
+        select.selectByValue(accountView);
+    }
+
+    /**
+     * Method to choose accounts view, if it is all accounts then press button go.
+     * @param accountView view.
+     * @param button to click.
+     */
+    public void chooseAccounts(final String accountView, final WebElement button) {
+        final int waitSeconds = 90;
+        WebDriverWait wait = DriverManager.getInstance().getWaiter();
+        WebElement searchOnDashboard = wait.until(ExpectedConditions.visibilityOfElementLocated(this.accountsViewList));
+        Select select = new Select(searchOnDashboard);
+        if ("All Accounts".equalsIgnoreCase(select.getFirstSelectedOption().getText().toString())) {
+            clickGoButton(button);
+        } else {
+            setAccountsViewList(accountView);
+        }
+    }
+
+    /**
+     * Method to get go button.
+     * @return button.
+     */
+    public WebElement getGoButton() {
+        return this.webDriver.findElement(this.goButton);
+    }
+
+    /**
+     * Method to click go button.
+     * @param button to click.
+     */
+    public void clickGoButton(final WebElement button) {
+        button.click();
+    }
+
+    /**
+     * Method to verify all accounts was selected.
+     * @param accountView list.
+     * @return list value.
+     */
+    public String isAllAccountsSelected(final String accountView) {
+        WebDriverWait wait = DriverManager.getInstance().getWaiter();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "wait" which hides the field declared at line 29. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)